.env.example

```diff
@@ -0,0 +1,6 @@
+OPENEDX_WEBHOOKS_CONFIG=development
+OAUTHLIB_INSECURE_TRANSPORT=1
+GITHUB_OAUTH_CLIENT_ID=
+GITHUB_OAUTH_CLIENT_SECRET=
+JIRA_OAUTH_CONSUMER_KEY=
+JIRA_OAUTH_RSA_KEY=
```

Comment:
An example environment file for Foreman to use. The idea is that a developer can copy `.env.example` to `.env`, put in real tokens (which are sensitive information, and should *not* be committed to the GitHub repository), and then run `foreman start` to run both the web process and the worker process. See also: http://ddollar.github.io/foreman/#ENVIRONMENT